css/legacy/includes/_impact.scss

```diff
@@ -608,3 +608,24 @@ i.fa-impact-manipulation {
 #impact_settings {
    display: none;
 }
+
+.target-popper {
+   display: none;
+   padding: 1rem;
+   text-align: left;
+   border-radius: 0;
+   border: 1px solid #ccc;
```

Comment:
About that, we should probably use some CSS variables.
@cconard96 do you have a hint?

Reply:
Should use --tblr-border-color probably